front/app/components/AssignBudgetControl/index.tsx

```diff
@@ -147,6 +147,9 @@ const AssignBudgetControl = memo(
             },
           });
           // if signed up & in
+        }
+        if (budgetingDisabledReason === 'not_active') {
```

Comment:
```suggestion
       } else if (budgetingDisabledReason === 'not_active') {
```

Both `isNilOrError(authUser)` and `budgetingDisabledReason === 'not_active'` can be true (if `authUser` is still pending), which means that without the `else` `openSignUpInModal` could be called twice